src/environmentd/tests/sql.rs

```diff
@@ -2337,24 +2337,28 @@ fn test_coord_startup_blocking() {
 }
 
 #[mz_ore::test]
-fn test_cancel_on_dropped_cluster() {
```

Comment:
Is it intentional that we don't have the test_cancel_on_dropped_cluster test anymore after this or does the test now cover both regressions?

Reply:
Before this PR, a peek on a dropped cluster would just hang forever and the user would have to manually cancel it. Now when the cluster is dropped all pending peeks are dropped immediately. So it's not possible for a user to cancel the peek on a dropped cluster.